### PR TITLE
[FIX] l10n_it_edi: SDI refuses lines with negative quantity

### PR DESCRIPTION
**Pull request by gawa-odoo**
Lines with negative quantity are refused by SDI.
To avoid this issue, we invert the quantity and price unit in this case.

opw-4402782

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by gawa-odoo** [comment]
I did not create a test, as it is done to unblock users.
But the resulting invoice xml does not really correspond to what we would really want (it needs to be functionally thought about)

File: addons/l10n_it_edi/models/account_move.py [hunk 1]
```diff
@@ -417,6 +417,15 @@ def _l10n_it_edi_grouping_function_total(self, base_line, tax_data):
         skip = tax_data['is_reverse_charge'] or self._l10n_it_edi_is_neg_split_payment(tax_data)
         return not skip
 
+    @api.model
+    def _inverse_negative_quantity_lines(self, base_lines):
```
Comment (epictete) [inline addons/l10n_it_edi/models/account_move.py:421]: Not sure having a dedicated method is useful, especially if we are only using it once :thinking: 
Maybe we could do something similar as what is already done for `reverse_charge_refund`?
Also, adding a comment explaining why we need to do the inversion might be nice :innocent:

**Review by epictete — APPROVED** [review]
Tanks for the quick update :+1: 

@robodoo r+

**Comment by epictete** [comment]
@robodoo r+

**Pull request closed without merging**